### PR TITLE
Add SingleQuoteSniff

### PR DESCRIPTION
**Pull request by dereuromark**
We have been using this for the last 3 years without a single incident. So quite safe to use - without false positives or alike.

4.2.0 minor

### CODE REVIEW

File: CakePHP/Sniffs/PHP/SingleQuoteSniff.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+namespace CakePHP\Sniffs\PHP;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Converts double quotes to single quotes for simple strings.
+ *
+ * @author Gregor Harlan <gharlan@web.de>
```
Comment (othercorey) [inline CakePHP/Sniffs/PHP/SingleQuoteSniff.php:22]: These are awkward. Does he wanted us to email him about it?

Comment (dereuromark) [reply CakePHP/Sniffs/PHP/SingleQuoteSniff.php:22]: Why? No need to.

Comment (othercorey) [reply CakePHP/Sniffs/PHP/SingleQuoteSniff.php:22]: r/whoosh

Comment (dereuromark) [reply CakePHP/Sniffs/PHP/SingleQuoteSniff.php:22]: ?

Comment (dereuromark) [reply CakePHP/Sniffs/PHP/SingleQuoteSniff.php:22]: Thats normal MIT license copy.

**Pull request merged**